ow_materials/src/MaterialDatabase.cpp

```diff
@@ -0,0 +1,26 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include <MaterialDatabase.h>
+
+using namespace ow_materials;
+
+void MaterialDatabase::addMaterial(Material mat) {
+  static MaterialID id = Material::id_min;
+  if (id == Material::id_max) {
+    // Unsure whether this should use gazebo or ROS logging, so both error
+    // message are left as TODOs for now
```

Comment:
Minor suggestion: use one of the logging methods for now, just so the error is reported.

Reply:
You're right. The redesign made it clear this is primarily a Gazebo plugin, so I will use Gazebo logging.